modules/account-lib/src/coin/dot/claimBuilder.ts

```diff
@@ -0,0 +1,118 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { DecodedSignedTx, DecodedSigningPayload, UnsignedTransaction } from '@substrate/txwrapper-core';
+import { methods } from '@substrate/txwrapper-polkadot';
+import BigNumber from 'bignumber.js';
+import { TransactionType } from '../baseCoin';
+import { InvalidTransactionError } from '../baseCoin/errors';
+import { ClaimArgs, MethodNames } from './iface';
+import { Transaction } from './transaction';
+import { TransactionBuilder } from './transactionBuilder';
+import { ClaimTransactionSchema } from './txnSchema';
+import { BaseAddress } from '../baseCoin/iface';
+import utils from './utils';
+
+export class ClaimBuilder extends TransactionBuilder {
+  protected _validatorStash: string;
+  protected _claimEra: string;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  /**
+   *
+   * Pay out all the stakers behind a single validator for a single era.
+   * Any account can create this transaction.
+   *
+   * @returns {UnsignedTransaction} an unsigned Dot transaction
+   *
+   * @see https://polkadot.js.org/docs/substrate/extrinsics/#payoutstakersvalidator_stash-accountid32-era-u32
+   */
+  protected buildTransaction(): UnsignedTransaction {
+    const baseTxInfo = this.createBaseTxInfo();
+    return methods.staking.payoutStakers(
+      {
+        validatorStash: this._validatorStash,
+        era: this._claimEra,
+      },
+      baseTxInfo.baseTxInfo,
+      baseTxInfo.options,
+    );
+  }
+
+  protected get transactionType(): TransactionType {
```

Comment:
nit: missing some jsdocs for some of the functions

Reply:
done